web_form_sticky_header/static/src/css/sticky_header.css

```diff
@@ -0,0 +1,8 @@
+.oe_form header {
+    position: sticky;
```

Comment:
I was not so sure about this one, but https://caniuse.com/#search=sticky says this is a thing. TIL, thanks!